[2025-06-18 16:25:02,754] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: connect_postgres_dag.create_postgres_table 2024-09-24T00:00:00+00:00 [queued]>
[2025-06-18 16:25:02,778] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: connect_postgres_dag.create_postgres_table 2024-09-24T00:00:00+00:00 [queued]>
[2025-06-18 16:25:02,778] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2025-06-18 16:25:02,779] {taskinstance.py:1043} INFO - Starting attempt 1 of 6
[2025-06-18 16:25:02,779] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2025-06-18 16:25:02,821] {taskinstance.py:1063} INFO - Executing <Task(PostgresOperator): create_postgres_table> on 2024-09-24T00:00:00+00:00
[2025-06-18 16:25:02,827] {standard_task_runner.py:52} INFO - Started process 2226 to run task
[2025-06-18 16:25:02,836] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'connect_postgres_dag', 'create_postgres_table', '2024-09-24T00:00:00+00:00', '--job-id', '15950', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/connect_postgres_dag.py', '--cfg-path', '/tmp/tmp_yfqz8vh', '--error-file', '/tmp/tmpm60_8gt_']
[2025-06-18 16:25:02,842] {standard_task_runner.py:77} INFO - Job 15950: Subtask create_postgres_table
[2025-06-18 16:25:02,943] {logging_mixin.py:104} INFO - Running <TaskInstance: connect_postgres_dag.create_postgres_table 2024-09-24T00:00:00+00:00 [running]> on host ee91c01336f6
[2025-06-18 16:25:03,024] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=connect_postgres_dag
AIRFLOW_CTX_TASK_ID=create_postgres_table
AIRFLOW_CTX_EXECUTION_DATE=2024-09-24T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-09-24T00:00:00+00:00
[2025-06-18 16:25:03,025] {postgres.py:69} INFO - Executing: 
            create table if not exists employee (
                "name" text,
                "age" integer,
                "salary" numeric
            )
        
[2025-06-18 16:25:03,041] {base.py:74} INFO - Using connection to: id: postgres_localhost. Host: postgres, Port: 5432, Schema: test, Login: airflow, Password: XXXXXXXX, extra: None
[2025-06-18 16:25:03,049] {dbapi.py:180} INFO - Running statement: 
            create table if not exists employee (
                "name" text,
                "age" integer,
                "salary" numeric
            )
        , parameters: None
[2025-06-18 16:25:03,050] {dbapi.py:186} INFO - Rows affected: -1
[2025-06-18 16:25:03,051] {postgres.py:73} INFO - NOTICE:  relation "employee" already exists, skipping

[2025-06-18 16:25:03,076] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=connect_postgres_dag, task_id=create_postgres_table, execution_date=20240924T000000, start_date=20250618T162502, end_date=20250618T162503
[2025-06-18 16:25:03,155] {taskinstance.py:1220} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-06-18 16:25:03,175] {local_task_job.py:146} INFO - Task exited with return code 0
